wiki/Gameplay/Sliderbreak/en.md

```diff
@@ -0,0 +1,18 @@
+---
+stub: true
+---
+
+# Sliderbreak
+
+<!-- not 100% sure about this but i always thought a sliderbreak was just losing FC from a slider, didnt know it had/(still has?) other meanings -->
+
+Sliderbreak has two meanings:
+
+- In [osu!standard](/wiki/Game_Modes/osu!):
+
+  - Tapping the slider-start too early (this can cause the slider to shake)
+  - Not collecting a slider tick (includes the hidden ones underneath the slider-start and slider-end).
+  - Releasing the slider ball before it hits the slider-end.
+    - This will not cause a miss, but instead will give the player a 100 with no combo increment.
+
+- The case where the player does not obtain the max possible combo without missing an object.
```

Comment:
```suggestion
In [osu!](/wiki/Game_Modes/osu!), a **sliderbreak** refers to losing combo by not collecting a slider tick (including the hidden one underneath the slider-start).

Sliderbreaks are often reffered to as **SB**.
```

or something like this

Reply:
do people call sliderbreaks SBs?

idk if its because im more into the mapping scene but first thing that comes to mind when i see sb is storyboard D:

Reply:
![image](https://user-images.githubusercontent.com/19843905/88467882-468d1700-cedc-11ea-8ca3-169eb853bc77.png)

i think? dunno about the letter capitalisation though